src/lsp/server.ml

```diff
@@ -34,7 +34,7 @@ let preprocess_uri uri =
   else uri
 
 let mk_prelude files =
-  List.map (
+  List.rev_map (
```

Comment:
why the `rev_map` here ?

Reply:
Because a `fold_left` is used in the `parse_settings` function, so the list of files is reversed, then re-reversed in `mk_prelude` to get back its order.

Reply:
Hm... in this case the lists will always be "short", so I'd rather we always keep the lists in the correct order even if that means less efficient traversals or a few more `List.rev`.

Reply:
True, I'll change it

Reply:
Done